edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -964,29 +915,30 @@ If you decide to modify the details of a competition, including the advancement
+If you still want to fill up some time and you have some longer events that have a cutoff, prefer increasing the cutoff time over increasing the time limit. This is because competitors don't often sign up for events which they do not expect to meet the time limit for. Combined with the fact that changing event registration is disallowed at most competitions, increasing the time limit results in neither significantly more opportunities for competitors nor takes up substantially more time. On the contrary, increasing the cutoff times often allows some competitors who are already signed up to have a better chance at getting a full mean or average, which consequently results in longer rounds. Use your experience with planning competitions to estimate a cutoff time that will extend the round by the desired amount.
 
-Other more dramatic options include increasing the size of a subsequent round by increasing the number of competitors who advance. You may also add an additional round entirely. The number of advancing competitors between rounds must still follow the maximum size as stated by [Regulation 9p1](https://www.worldcubeassociation.org/regulations/#9p1). In both cases, keep in mind that some competitors may have already left the venue if they were not expecting to advance to a certain round. Unless you have a good reason to expect that the majority of the competitors who could take advantage of modified advancement conditions will still be present (e.g. a large 2nd round of a popular event like 3x3x3 still ahead of you in the schedule), generally avoid changing advancement conditions.
+Other more dramatic options include increasing the number of competitors who advance to a subsequent round. You may also add an additional round entirely. The number of advancing competitors between rounds must still follow the maximum size as stated by [Regulation 9p1](https://www.worldcubeassociation.org/regulations/#9p1). In both cases, keep in mind that some competitors may have already left the venue if they were not expecting to advance to a subsequent round. Unless you have a good reason to expect that the majority of the competitors who could take advantage of modified advancement conditions will still be present (e.g. a large 2nd round of a popular event like 3x3x3 still ahead of you in the schedule), generally avoid changing advancement conditions.
 
-Per point 7.4 of the WCRP announcement criteria, you may not add an event to an ongoing WCA competition, except as an unofficial event.
+As per point 7.4 of the WCRP announcement criteria, you may not add an event to an ongoing WCA competition, except as an unofficial event, for which you would have to use a third-party service.
```

Comment:
I think it would make more sense to have a section on running unofficial events than it would be to describe a single point about running unofficial events here.

Reply:
I think unofficial stuff is a bit out-of-scope here. I only added this for clarity that those rounds would NOT be held through wca/wca live

Reply:
Makes sense.  I guess for the most part that's the only real significant difference in how they're ran.